src/help/technicalHistory.md

```diff
@@ -0,0 +1,48 @@
+#### January 18, 2017
```

Comment:
If we already have a history.md file, why does this file have the same contents? Do we now have 3 history files? 2 `history.md` files, one in `src/help` and one in `public/help` and then this `technicalHistory.md`?